### DISCUSSION TITLE
romeerez is god and orchid-orm is masterpiece

### DISCUSSION THREAD

**Discussion by minkoonim** [romeerez/orchid-orm#240] [General]
long live the orchid

I just have finished reading the doc,
and I just had to write here how great your library is

Thanks for great work

**Comment by romeerez** [comment]
I am just a simple man working on the code, and Orchid is great - loving it as well, thank you!

**Reply by minkoonim** [reply]
any plan on automating migration processes?
I mean orchid's 
benchmark - good
orm system - best I've ever seen
migration - ...

You fix the migration,
I really think you can attract thousands of people
orchid is a lot better than drizzle

**Reply by romeerez** [reply]
Do you mean to make it work like in Drizzle?

Yes, going to make it, it's a big one so I was delaying it for the future, maybe it will be in a couple of months or so.

An interesting fact about Drizzle's migrations - they're the closed source, drizzle-kit isn't on GitHub, so it can't be borrowed :)

**Reply by minkoonim** [reply]
well drizzle isn't the first one to do it anyway
I mean python's Django from decades ago also have same detecting new changes and making migration files.
it's open source maybe you can check theirs

**Reply by romeerez** [reply]
@minkoonim auto-migrations are ready!

[docs](https://orchid-orm.netlify.app/guide/orm-and-query-builder.html#generate-migrations)

**Reply by minkoonim** [reply]
hell yeah!!!!!!!!!! great work man  I love it so much Thank you!!

**Reply by romeerez** [reply]
Thank you! That was an incredible amount of work, so I hope it will save you and other people a decent amount of time.

[in the PR](https://github.com/romeerez/orchid-orm/pull/248) it's +20,496 −11,378 changes.

I tried it out on my demo app, and it felt fantastic.

**Comment by romeerez** [comment]
Closing, as someone may read the title and think it's a cult or something :)
